pry-doc.gemspec

```diff
@@ -23,7 +23,7 @@ DESCR
   s.required_ruby_version = '>= 2.0'
 
   s.add_dependency 'yard', "~> 0.9"
-  s.add_dependency 'pry', "~> 0.9"
+  s.add_dependency 'pry', "~> 0.9.11"
```

Comment:
Looks like you wanted to bump yard, but bumped pry by accident 😁

Reply:
Just fixed it https://github.com/pry/pry-doc/pull/81/commits/169f9ebaa8152845aa90f29669d9d07e5ac511ec